spec/card_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+describe Card do
+
+    it 'should compare two words and add error if they equal' do
```

Comment:
Use 2 (not 4) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.